go.mod

```diff
@@ -5,6 +5,7 @@ go 1.23
 require (
 	github.com/edwarnicke/exechelper v1.0.2
 	github.com/edwarnicke/log v1.0.0
+	github.com/networkservicemesh/sdk v1.14.1
```

Comment:
I'm not sure that we need to add this dependency, because in this case we'll need to include vpphelper for releasing/updating from SDK.